docs/GettingStartedDocs/Contributors/NonAccMachineSGXLinuxGettingStarted.md

```diff
@@ -0,0 +1,229 @@
+
+## 2. Set up openenclave environment
+### 2.1 Clone Open Enclave SDK repo from GitHub
+Use the following command to download the source code.
+
+```bash
+git clone https://github.com/openenclave/openenclave.git
+```
+
+This creates a source tree under the directory called openenclave.
+
+### 2.2 Install project requirements
+
+First, change directory into the openenclave repository:
+```bash
+cd openenclave
+```
+
+Ansible is required to install the project requirements. If not already installed, you can install it by running:
+```bash
+sudo scripts/ansible/install-ansible.sh
+```
+
+Then run the following command to install the dependency:
+```bash
+ansible-playbook scripts/ansible/oe-contributors-setup.yml
+```
+NOTE: The Ansible playbook commands from above will try to execute tasks with sudo rights. Make sure that the user running the playbooks has sudo rights, and if it uses a sudo password add the following extra parameter --ask-become-pass.
+
+## 3. Set up Intel DCAP Quote Provider Library (QPL):
+### 3.1 Install Intel DCAP Quote Provider by using its Debian package
+The [libsgx-dcap-default-qpl directory](https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/ubuntuServer18.04/debian_pkgs/libs/libsgx-dcap-default-qpl/) lists all different version of libsgx-dcap-default-qpl, please download the most recent version that matches your OS version. For Ubuntu 18.04 (code name [Bionic Beaver](https://wiki.ubuntu.com/BionicBeaver)), please download the version libsgx-dcap-default-qpl_{version}-bionic1_amd64.deb. For Ubuntu 16.04 (code name [Xenial Xerus](https://wiki.ubuntu.com/XenialXerus)), please download the version libsgx-dcap-default-qpl_{version}-xenial1_amd64.deb.
+
+In this document, we use libsgx-dcap-default-qpl_1.6.100.2-bionic1_amd64.deb as an example, run the command below to download the package
+```bash
+cd ~
+wget https://download.01.org/intel-sgx/sgx-dcap/1.6/linux/distro/ubuntuServer18.04/debian_pkgs/libs/libsgx-dcap-default-qpl/libsgx-dcap-default-qpl_1.6.100.2-bionic1_amd64.deb
+```
+
+Then install the package
+```bash
+sudo dpkg -i libsgx-dcap-default-qpl_1.6.100.2-bionic1_amd64.deb
+```
+
+### 3.2 Create a soft link
+OE expects the file name of the qpl to be libdcap_quoteprov.so. But the Intel default qpl creates installed libdcap_quoteprov.so.1 and libdcap_quoteprov.so.1.6.100.2.  libdcap_quoteprov.so.1 is a soft link to libdcap_quoteprov.so.1.6.100.2. To allow OE works properly, we need to create the other soft link called libdcap_quoteprov.s linking to libdcap_quoteprov.so.1.6.100.2
+
+Check where those files are installed.
+```bash
+dpkg --listfiles libsgx-dcap-default-qpl
+```
+
+In most cases, it should be in /usr/lib/x86_64-linux-gnu/
+
+Use /usr/lib/x86_64-linux-gnu/ as an example.
+```bash
+cd /usr/lib/x86_64-linux-gnu/
+sudo ln -s libdcap_quoteprov.so.1.6.100.2 libdcap_quoteprov.so
```

Comment:
We should probably request that intel create this file when installing the packge. Not blocking for this PR though.

Reply:
Thanks! Yes, that's definitely the direction we want to go. But I am not sure if it should be Intel or Azure Security to change the library name to meet the other's standard at this moment. I opened an issue at Intel DCAP's repo to start the conversation for this matter.  https://github.com/intel/SGXDataCenterAttestationPrimitives/issues/104